cylc/flow/cycling/iso8601.py

```diff
@@ -668,13 +665,14 @@ def ingest_time(value: str, now: Optional['TimePoint'] = None) -> str:
 
     # correct for year in 'now' if year only,
     # or year and time, specified in input
+        # TODO: Figure out why this correction is needed
```

Comment:
Should this still be here? If yes, is there a matching issue?

Reply:
Not sure if it's worth an issue, whatever it's doing it seems to be working. But if someone can figure this out down the line it would be good to update the comment above to improve the explanation

Reply:
I've spent far too long trying to figure this out - I think I'd quite like an issue raised to poke this - I'm not thrilled that I've not been able to get the bottom of it.

I'm also not clear that I can find definitive docs of what `--icp` can accept.

Reply:
>[oliver-sanders](https://github.com/oliver-sanders) [on 16 Nov 2018](https://github.com/cylc/cylc-flow/pull/2815#discussion_r234184335)
>Comments like this have a habit of becoming a bit cryptic to future developers. Perhaps copy in the URL of the isodatetime issue.

https://github.com/cylc/cylc-flow/pull/2815#discussion_r234184335

Sadly not done for this particular comment 😬

Reply:
I have opened 2 issues:
- https://github.com/cylc/cylc-flow/issues/4805
- https://github.com/cylc/cylc-doc/issues/455

Reply:
> I'm also not clear that I can find definitive docs of what --icp can accept.

This. Yes. #4807 (Put in `cylc-flow` not `cylc-doc` because although it should touch both I think the CLI help is upstream)